<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Screenshare</class>
 <widget class="QDialog" name="Screenshare">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="mylabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>290</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>320</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>FPS</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="lineWrapMode">
    <enum>QPlainTextEdit::WidgetWidth</enum>
   </property>
   <property name="plainText">
    <string>10</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>320</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ultrafast</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>superfast</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>veryfast</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>faster</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>fast</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>medium</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>slow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>slower</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>baseline</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>main</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>high</string>
    </property>
   </item>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mylabel</class>
   <extends>QLabel</extends>
   <header>mylabel.h</header>
   <slots>
    <signal>clicked()</signal>
    <signal>doubleclicked()</signal>
    <signal>keypressed()</signal>
    <signal>resizeevent()</signal>
    <slot>slotClicked()</slot>
    <slot>slotDoubleClicked()</slot>
    <slot>slotKeyPressed()</slot>
    <slot>slotresizeEvent()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
